CorePluginFramework/src/au/gov/asd/tac/constellation/plugins/gui/ValueInputPane.java

```diff
@@ -294,18 +294,21 @@ protected void updateItem(final String item, final boolean empty) {
     @Override
     public void recentValuesChanged(final RecentValuesChangeEvent e) {
         if (recentValuesCombo != null && parameterId.equals(e.getId())) {
-            recentValuesCombo.getSelectionModel().selectedIndexProperty().removeListener(recentValueSelectionListener);
-            final List<String> recentValues = e.getNewValues();
-            if (recentValues != null) {
-                recentValuesCombo.setItems(FXCollections.observableList(recentValues));
-                recentValuesCombo.setDisable(false);
-            } else {
-                final List<String> empty = Collections.emptyList();
-                recentValuesCombo.setItems(FXCollections.observableList(empty));
-                recentValuesCombo.setDisable(true);
-            }
-            recentValuesCombo.setPromptText("...");
-            recentValuesCombo.getSelectionModel().selectedIndexProperty().addListener(recentValueSelectionListener);
+            //#1608: JavaFX code allows updating the UI from an JavaFX application thread.
```

Comment:
Minor but are you able to omit the ticket number in this comment? Just thinking that it might be a bit confusing if you don't have the context that this number is a ticket number (I think removing will be better than clarifying the context)

Reply:
Done